From 0227ca47cfa0e470d4b165f950b373416774af9d Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Thu, 30 May 2019 11:00:30 +0200
Subject: [PATCH 1/3] fs: fat: use if FS_FAT in Kconfig

Use `if FS_FAT` to indent dependent FAT options in Kconfig.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 fs/fat/Kconfig | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/fs/fat/Kconfig b/fs/fat/Kconfig
index 9bb11eac9f..12524eeeed 100644
--- a/fs/fat/Kconfig
+++ b/fs/fat/Kconfig
@@ -6,9 +6,10 @@ config FS_FAT
 	  It is useful mainly for its wide compatibility with various operating
 	  systems. You can also enable CMD_FAT to get access to fat commands.
 
+if FS_FAT
+
 config FAT_WRITE
 	bool "Enable FAT filesystem write support"
-	depends on FS_FAT
 	help
 	  This provides support for creating and writing new files to an
 	  existing FAT filesystem partition.
@@ -16,9 +17,10 @@ config FAT_WRITE
 config FS_FAT_MAX_CLUSTSIZE
 	int "Set maximum possible clustersize"
 	default 65536
-	depends on FS_FAT
 	help
 	  Set the maximum possible clustersize for the FAT filesytem. This
 	  is the smallest amount of disk space that can be used to hold a
 	  file. Unless you have an extremely tight memory memory constraints,
 	  leave the default.
+
+endif
-- 
2.20.1

